docs/release-notes/self-hosted/self-hosted-v12.0.0.md

```diff
@@ -0,0 +1,128 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v12.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.1.6
+codacy_tools_version_new: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.2.2
+---
+
+# Self-hosted v12.0.0
+
+These release notes are for [Codacy Self-hosted v12.0.0](https://github.com/codacy/chart/releases/tag/12.0.0){: target="_blank"}, released on June 21, 2023.<!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v12.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v12.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the following versions:
+
+    -   [Codacy Analysis CLI 7.8.3](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.8.3)
+    -   [Codacy Coverage Reporter 13.13.1](https://github.com/codacy/codacy-coverage-reporter/releases/tag/13.13.1)
+
+## Breaking changes
+
+-   If you have set `scheduler.enable: "true"` in `values.yaml`, do the following before updating Kubernetes.
+
+    Assuming you're using the default `codacy` namespace:
+
+    1.  Remove all `scheduler.*` settings from `values.yaml`
```

Comment:
@tfrgoncalves Can you validate? See [Ranga's comment](https://github.com/codacy/docs/pull/1767#discussion_r1267027931) for more context.

cc: @Ngwerume


```suggestion
-   If you have set `scheduler.enable: "true"` in `values-production.yaml`, do the following before updating Kubernetes.

    Assuming you're using the default `codacy` namespace:

    1.  Remove all `scheduler.*` settings from `values-production.yaml`
```

Reply:
Considering user overrides should be done in values-production.yaml and values-microk8s.yaml both should probably be mentioned. @lolgab @Ngwerume, do you agree?

Reply:
As it's [written on the installation docs](https://github.com/codacy/chart/blob/082dab58d173ca5824a2d9994f7d979b9a414eb3/docs/index.md#L80), `values-microk8s.yaml` can be used instead of `values-production.yaml` so you might want to add a notice saying that the changes are to be applied to `values-microk8s.yaml` if your are using MicroK8s.

Reply:
Thanks @lolgab ! Here the updated suggestion:

cc: @Ngwerume @tfrgoncalves 

```suggestion
-   If you have set `scheduler.enable: "true"` in `values-production.yaml` (or `values-microk8s.yaml` if using MicroK8s), do the following before updating Kubernetes.

    Assuming you're using the default `codacy` namespace:

    1.  Remove all `scheduler.*` settings from `values-production.yaml` (or `values-microk8s.yaml` if using MicroK8s)
```